content/SCALE/SCALETutorials/Storage/ManagePoolsScale.md

```diff
@@ -54,32 +56,148 @@ Select **Confirm Export/Disconnect**
+
+The upgrade itself only takes a few seconds and is non-disruptive.
+It is not necessary to stop any sharing services to upgrade the pool.
+However, the best practice is to upgrade when the pool is not in heavy use.
+The upgrade process suspends I/O for a short period but is nearly instantaneous on a quiet pool.
+
+## Running a Pool Data Integrity Check (Scrub)
+
+Use **Scrub** on the **ZFS Health** pool widget to start a pool data integrity check.
+
+{{< trueimage src="/images/SCALE/Storage/StorageDashboardZFSHealthWidget.png" alt="ZFS Health Widget" id="ZFS Health Widget" >}}
+
+Click **Scrub** to open the **Scrub Pool** dialog.
+Select **Confirm**, then click **Start Scrub**.
+
+If TrueNAS detects problems during the scrub operation, it either corrects them or generates an [alert]({{< relref "/SCALE/SCALEUIReference/TopToolbar/Alerts/_index.md" >}}) in the web interface.
+
+By default, TrueNAS automatically checks every pool on a recurring scrub schedule.
+
+The **ZFS Health** widget displays the state of the last scrub or disks in the pool.
+To view scheduled scrub tasks, click **View all Scrub Tasks** on the **ZFS Health** widget.
+
+## Managing Pool Disks
+
+The **Storage Dashboard** screen **Disks** button and the **Manage Disks** button on the **Disk Health** widget both open the **Disks** screen.
+
+**Manage Devices** on the **Topology** widget opens the ***Poolname* Devices** screen.
+To manage disks in a pool, click on the VDEV to expand it and show the disks in that VDEV.
+Click on a disk to see the devices widgets for that disk.
+You can take a disk offline, detach it, replace it, manage the SED encryption password, and perform other disk management tasks from this screen.
+
+See [Replacing Disks]({{< relref "ReplacingDisks.md" >}}) for more information on the **Offline**, **Replace** and **Online** options.
+
+## Expanding a Pool
+
+You can increase the size of an existing pool by adding one or more drives to an existing RAIDZ VDEV, adding a new VDEV of the same type, or by replacing all existing disks in the VDEV with larger disks.
+
+By default, a VDEV limits all disks to the usable capacity of the smallest attached device.
+TrueNAS SCALE automatically expands the usable capacity of the pool to fit all available space once the smallest attached disk is replaced (see [Replacing Disks to Expand a Pool](#replacing-disks-to-expand-a-pool) below).
+If a pool is not automatically expanded, for example when resizing virtual disks in a hypervisor apart from TrueNAS, click **Expand** on the **Storage Dashboard** to manually increase the pool size to match all available disk space.
+
+### Extending a RAIDZ VDEV
+
+Additional disks can be added one at a time to a RAIDZ VDEV, expanding its capacity incrementally.
+This can be useful for small pools (typically with only one RAID-Z VDEV), where there isn't sufficient hardware capacity to add a mirrored VDEV by doubling the number of disks.
```

Comment:
```suggestion
This can be useful for small pools (typically with only one RAID-Z VDEV), where there isn't sufficient hardware capacity to add a second VDEV, doubling the number of disks.
```